synapse/storage/events.py

```diff
@@ -2330,6 +2324,16 @@ def _purge_history_txn(
             ]
         )
 
+        # the event peristence loop takes out an exclusive lock on room_depth
```

Comment:
i stared at this stupidly thinking that "event persistence loop" meant the loop immediately above which is purging and persisting the new events... rather than the general BAU activity of persisting events. 

If you can clarify that you're talking about "synapse takes out an exclusive lock on room_depth whenever it persists events..." it would avoid confusing bears of very little brain such as I